gapic/src/main/com/google/gapid/views/FramebufferView.java

```diff
@@ -128,14 +139,37 @@ public FramebufferView(Composite parent, Models models, Widgets widgets) {
 
     setLayout(new GridLayout(2, false));
 
-    attachmentListener = new AttachmentListener(widgets.theme);
+    Composite header = createComposite(this, new GridLayout(2, false));
 
-    toolBar = createToolBar(widgets.theme);
+    picker = new AttachmentPicker(this);
+    picker.addContentListener(SWT.MouseDown,
+      e -> picker.selectAttachment(picker.getItemAt(e.x)));
+
+    ToolBar toolBar = createToolBar(widgets.theme);
     imagePanel = new ImagePanel(this, View.Framebuffer, models.analytics, widgets, true);
 
+    GridData pickerGridData = new GridData(SWT.FILL, SWT.FILL, true, true, 2, 1);
```

Comment:
Would it be better to make it `new GridData(SWT.FILL, SWT.FILL, true, false, 2, 1);`? It will avoid blank area between the picker strip and the framebuffer below.

Reply:
My worry with this is that now it's too small in some cases. I'm looking into having it resize or use a sashform.